### PR TITLE
Twilio

### PR DESCRIPTION
**Pull request by lemingos**

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* [Creating an SMS Notification](/tutorials/notifications/creating-sms-notification
+* [Twilio API Reference, REST API](https://www.twilio.com/docs/usage/api)
+* [Managing Integrations in Partner Portal](/tutorials/partner-portal/managing-integrations)
+## Steps
+
+Integrating SMS service with Twilio using SMS notifications is a three-step process:
+
+1. Create Twilio Account Using Partner Portal
+2. Add SMS notification to the form
+3. Billing
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]: https://github.com/mdyd-dev/nearme-documentation/pull/385#issue-251470321

Can you use automagic? ;)

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Step 1: Create Twilio Account Using Partner Portal
+
+As a first step you need to create Twilio account that will be associated with your instance. To do that, open your instance view in Partner Portal and follow with "Integrations" link on the top.
+At the very bottom of Integrations page you will see `Enable Twilio` button. When you click it, and confirm with next "Enable Twilio Integration" button, Partner Portal will create a subaccount in platformOS Twilio account that will be coneccted to your instance.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]:
```suggestion
At the very bottom of Integrations page you will see `Enable Twilio` button. When you click it, and confirm with next "Enable Twilio Integration" button, Partner Portal will create a sub-account in platformOS Twilio account that will be connected to your instance. 
```

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+As a first step you need to create Twilio account that will be associated with your instance. To do that, open your instance view in Partner Portal and follow with "Integrations" link on the top.
+At the very bottom of Integrations page you will see `Enable Twilio` button. When you click it, and confirm with next "Enable Twilio Integration" button, Partner Portal will create a subaccount in platformOS Twilio account that will be coneccted to your instance. 
+
+At this point you should see the notification: "We're preparing twilio integration. Refresh page in a few minutes."
+And everything should be ready within a few seconds.
+
+### Step 2: Add SMS notification to your forms
+
+This process is explained in detailed in [Creating an SMS Notification](/tutorials/notifications/creating-sms-notification)
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]:
```suggestion
This process is explained in details in [Creating an SMS Notification](/tutorials/notifications/creating-sms-notification)
```

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Step 3: Billing
+
+Based on the usage recorded in your Twilio account, platfromOS will charge you on the monthly basis and the summary of your Twilio costs will be added to your monthly invoice. Please note that we add 15% commision to all Twilio costs.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]:
```suggestion
Based on the usage recorded in your Twilio account, platformOS will charge you on the monthly basis and the summary of your Twilio costs will be added to your monthly invoice. Please note that we add 15% commission to all Twilio costs.
```

Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]: Thank you! Fixed in my review.

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid (hunk 3 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]: Thank you, fixed in my review.

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid (hunk 2 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]: Thank you, fixed.

File: marketplace_builder/views/pages/tutorials/integrations/twilio.liquid (hunk 1 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/integrations/twilio.liquid]: Added during review! Thank you!

**Comment by diana-lakatos** [comment]
Reviewed and edited.

**Pull request merged**